### PR TITLE
ci: use `Tui` prefix for exported classes

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [x] Build or CI related changes
- [ ] Documentation content changes

### CODE REVIEW

File: projects/addon-editor/components/toolbar-tools/code/code.component.ts [hunk 1]
```diff
@@ -38,3 +43,10 @@ export class CodeComponent {
         }
     }
 }
+
+@Component({
+    selector: 'tui-code',
```
Comment (waterplea) [inline projects/addon-editor/components/toolbar-tools/code/code.component.ts:48]: Two components cannot match the same selector. I think we can just rename the class.

File: projects/addon-doc/src/directives/scroll-into-view/scroll-into-view.directive.ts [hunk 2]
```diff
@@ -30,3 +35,9 @@ export class ScrollIntoViewDirective {
             });
     }
 }
+
+@Directive({
+    selector: '[tuiScrollIntoView]',
+    providers: [TuiDestroyService],
+})
+export class TuiScrollIntoViewDirective extends ScrollIntoViewDirective {}
```
Comment (waterplea) [inline projects/addon-doc/src/directives/scroll-into-view/scroll-into-view.directive.ts:43]: Add to migrations

File: projects/addon-preview/components/preview-dialog/preview-dialog.service.ts [hunk 3]
```diff
@@ -18,3 +20,6 @@ export const PREVIEW_DIALOG_PROVIDER: Provider = {
     useExisting: PreviewDialogService,
     multi: true,
 };
+
+@Injectable({providedIn: 'root'})
+export class TuiPreviewDialogService extends PreviewDialogService {}
```
Comment (waterplea) [inline projects/addon-preview/components/preview-dialog/preview-dialog.service.ts:25]: Add to migrations if it's missing there

File: projects/cdk/exceptions/invalid-day.exception.ts [hunk 4]
```diff
@@ -1,5 +1,12 @@
+/**
+ * @deprecated: use {@link TuiInvalidDayException}
+ * TODO: remove in v3.0
+ */
+// eslint-disable-next-line @typescript-eslint/naming-convention
 export class InvalidDayException extends Error {
```
Comment (waterplea) [inline projects/cdk/exceptions/invalid-day.exception.ts:6]: Migrations for these 3 too

**Review by waterplea — APPROVED** [review]

File: projects/addon-editor/components/toolbar-tools/code/code.component.ts (hunk 1 above)
Comment (splincode) [reply projects/addon-editor/components/toolbar-tools/code/code.component.ts]: fixed

File: projects/addon-doc/src/directives/scroll-into-view/scroll-into-view.directive.ts (hunk 2 above)
Comment (splincode) [reply projects/addon-doc/src/directives/scroll-into-view/scroll-into-view.directive.ts]: fixed

File: projects/addon-tablebars/classes/table-bar.ts [hunk 5]
```diff
@@ -27,3 +32,5 @@ export class TableBar {
         this.observer.complete();
     }
 }
+
+export class TuiTableBar extends TableBar {}
```
[the remark this answers is not in the document]
Comment (splincode) [reply projects/addon-tablebars/classes/table-bar.ts:36]: fixed

File: projects/kit/components/data-list-wrapper/data-list-wrapper.ts [hunk 6]
```diff
@@ -56,3 +61,6 @@ export abstract class AbstractDataListWrapper<T> {
             .filter(isPresent);
     }
 }
+
+@Directive()
+export abstract class TuiAbstractDataListWrapper<T> extends AbstractDataListWrapper<T> {}
```
[the remark this answers is not in the document]
Comment (splincode) [reply projects/kit/components/data-list-wrapper/data-list-wrapper.ts]: fixed

**Comment by splincode** [comment]
@waterplea @vladimirpotekhin @nsbarsukov hello, please review me later)

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**